exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/HistoryEntity.java

```diff
@@ -0,0 +1,119 @@
+package com.exonum.binding.cryptocurrency;
+
+import com.exonum.binding.common.crypto.PublicKey;
+import com.exonum.binding.common.hash.HashCode;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+public class HistoryEntity {
```

Comment:
Why don't we use the protobuf-generated message — with it 
1. Serializer impl will be a one-liner (and gone with reflective serializer impl later).
2. Builders are free.

?